pkg/bootstrap/authenticate.go

```diff
@@ -22,7 +22,7 @@ import (
 
 // Authenticator generates authentication credentials for requests.
 type Authenticator interface {
-	CreateToken(body []byte) (string, error)
+	CreateToken(body []byte, jwtToken string) (string, error)
```

Comment:
nit: the "T" in "JWT" stands for "Token", so "jwtToken" is redundant.

```suggestion
	CreateToken(body []byte, jwt string) (string, error)
```